packages/frontend-vital-signs/sonar-project.properties

```diff
@@ -0,0 +1,19 @@
+sonar.projectKey=govuk-one-login_frontend-vital-signs
+sonar.organization=govuk-one-login
+
+# This is the name and version displayed in the SonarCloud UI.
+sonar.projectName=frontend-vital-signs
+sonar.projectVersion=1.0
+
+# Language-specific settings
+sonar.language=typescript
+
+sonar.javascript.lcov.reportPaths=coverage/lcov.info
+
+sonar.sources=src/
+sonar.exclusions=**/*.test.ts,**/*.spec.ts,coverage/**
+
+sonar.tests=src/
```

Comment:
Ah so in this repo the sonar scanner runs relative to the location of this `sonar-project.properties` file, so it will pick up the `src/` folder for this package.